utils/_context/_scenarios/__init__.py

```diff
@@ -751,6 +751,20 @@ def all_endtoend_scenarios(test_object):
         scenario_groups=[ScenarioGroup.DEBUGGER],
     )
 
+    debugger_exception_replay = EndToEndScenario(
+        "DEBUGGER_EXCEPTION_REPLAY",
+        rc_api_enabled=True,
+        weblog_env={
+            "DD_DYNAMIC_INSTRUMENTATION_ENABLED": "1",
+            "DD_REMOTE_CONFIG_ENABLED": "true",
+            "DD_EXCEPTION_REPLAY_ENABLED": "true",
```

Comment:
Would it be possible to merge this scenario to an existing debugger scenario ?

Reply:
I prefer not. `Exception Replay` is feature, where we are using code that intentionally throws errors and we validate it. It will be full with many different complex scenarios, and I really don't want this to be mixed with normal behaviour.